### ISSUE TITLE
[Bug]: Stall doors not working properly

### ISSUE THREAD

**Issue by monobanana1** [AlaharranHonor/SWEquestrianM#122]

**Comment by monobanana1** [comment]
Yes, I have a modpack with 300+ mods, that's probably the problem, but I don't have any mod that modifies the behavior of the doors.

I'm going to try to remove all mods and leave only Swem installed.

**Comment by Scope213** [comment]
Hi, i know this is an old post but i was wondering if a fix has been found?

if yes please DM me on Discord since i dont use github much
Scope#4431

**Comment by XDelphix** [comment]
To my understanding no fix has been found, and stall doors appear working on all SWEM servers and SP and with 300 (unknown) mods there is a lot of room for potential incompatibility.
